content/chronograf/v1.8/about_the_project/release-notes-changelog.md

```diff
@@ -8,6 +8,26 @@ menu:
     parent: About the project
 ---
 
+## v1.8.9 [2020-12-04]
+
+### Features
+- Configure etcd with client TLS certificate.
+- Support Flux in InfluxDB Cloud and InfluxDB OSS 2.x sources.
+- Upgrade Go to 1.15.5.
+- Upgrade Node.js to 14 LTS.
+
+### Bug Fixes
```

Comment:
2x check this please.  I think a couple of small changes landed in the last 24hrs.